### ISSUE TITLE
Update Project Profile: Food Oasis (Fix Hannah Zulueta's Slack link)

### ISSUE THREAD

**Issue by JessicaLucindaCheng** [hackforla/website#6317]
### Prerequisites
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started) and attend an onboarding session.
2. You have already read our [How to Contribute to Hack for LA Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md).

### Overview
We need to fix the link to Hannah Zulueta's Slack profile so that it resolves to Hannah's active account.

### Action Items
- [x] In your IDE, open the `_projects/food-oasis.md` file
- [x] Observe the existing syntax of the front matter block [^1] in the file
- [x] Under the `leadership` section, find Hannah Zulueta's profile information.
- [x] Then, change Hannah Zulueta's Slack link
  **From**
  ```
        slack: "https://hackforla.slack.com/team/U01G7AH18J3"
  ```
  **To**
  ```
        slack: "https://hackforla.slack.com/team/U9SCMTNK0"
  ```
- [x] Verify the changes by viewing the following in your local environment:
  - [x] Food Oasis page [^2]
- [x] Once your pull request is merged, go to #6313  and
  - [x] Check off this issue under the _Dependency_ section
  - [x] If all the issues in the _Dependency_ section are checked off, move #6313 to the **New Issue Approval** column and uncheck the **Dependency label**.
    - <details>
        <summary>Click here to see how to uncheck the <b>Dependency label</b></summary>
        <img src="https://github.com/hackforla/website/assets/31293603/6f53f4d4-7d2c-45f8-8534-9936fc9adee8" width="300px">
      </details>

### Resources/Instructions
[^1]: [Info about the front matter block](https://jekyllrb.com/docs/front-matter/)
[^2]: Project detailed info page URL: https://www.hackforla.org/projects/food-oasis

**Comment by luisitocanlas** [comment]
Availability: 2/20/24 ~ 2/21/24
ETA: EOD 2/21/24 earliest or EOD 2/22/24 latest
